### PR TITLE
[KUBOS-422] KubOS Linux on iOBC updates

### PR DESCRIPTION
**Pull request by CatFreed**
Updated the linux on iobc doc:

* Updated .dtb NOR flash address
* Added blurbs about upgrade and recovery
* Added instructions for using the new SD card formatting script
* Fixed misc formatting problems from Doxygen

### CODE REVIEW

File: docs/kubos-linux-on-iobc.md [hunk 1]
```diff
@@ -119,7 +145,32 @@ You should see a device that looks like this:
     
 In this example '/dev/sdb' is the name of the SD card.  You might also see '/dev/mmcblk0'.  You'll need to use this name in all future commands.
 
-####Partition the SD Card
+#### Run the Formatting/Flashing Script
+
+Navigate to your 'kubos-linux-build' folder and open the 'tools' directory.
+
+Run the `format-sd.sh` script. You might need to run as root to get permissions for certain steps.
```
Comment (plauche) [inline docs/kubos-linux-on-iobc.md]: Is this script run from the `kubos-linux-build` directory or the `tools` directory?

File: docs/kubos-linux-on-iobc.md [hunk 2]
```diff
@@ -160,11 +227,11 @@ Mount the partitions
     $ sudo mount {name}{partition5} boot
```
Comment (plauche) [inline docs/kubos-linux-on-iobc.md]: I think these name/partition instructions might make a little more sense if we just picked a device, like `/dev/sdb`, and used it throughout the instructions. The `{name}{partition5}` part of the command `sudo mount {name}{partition5} boot` looks a bit more complex and confusing than `/dev/sdb5`.

File: docs/kubos-linux-on-iobc.md [hunk 3]
```diff
@@ -207,7 +274,7 @@ auto-boot and bring up the CLI.
 
 ![U-Boot Console](images/iOBC/uboot_console.png)
 
-####Flash the Files
+#### Flash the Files
 
 Start up SAM-BA.  You'll want to select the at91sam9g20-ISISOBC option from the 'Select your board' drop-down.
```
Comment (plauche) [inline docs/kubos-linux-on-iobc.md]: Some of the images used here are quite a bit bigger than their content and can probably be cropped down.

File: docs/kubos-linux-on-iobc.md [hunk 4]
```diff
@@ -246,3 +313,23 @@ baudrate of 115200.
...
+
+If you already have KubOS Linux installed on your system, but would like to upgrade to the latest version, check out the 'Upgrade Installation' section of the [KubOS Linux Upgrade doc](docs/kubos-linux-upgrade.md). Alternatively, if you would like to rollback to a previously installed version, refer to the 'Upgrade Rollback' section of the same document.
+
+## Recovery Process {#recovery-process}
+
+Should your KubOS Linux kernel become corrupted (as indicated by failing to successfully boot into Linux several times), the system will automatically try to recover during the next boot.  
+
+It will go through the following steps, if each is present:
+
+1. Reload the current version of KubOS Linux from the kpack*.itb file in the upgrade partition
```
Comment (plauche) [inline docs/kubos-linux-on-iobc.md]: Reload implying a certain number of boot attempts before reloading the next available OS option?

File: docs/kubos-linux-on-iobc.md (hunk 1 above)
Comment (CatFreed) [inline docs/kubos-linux-on-iobc.md]: The tools directory. Is the wording in the previous sentence bad?

File: docs/kubos-linux-on-iobc.md (hunk 4 above)
Comment (CatFreed) [inline docs/kubos-linux-on-iobc.md]: yes, the system will attempt to boot again after reloading the current OS (and after each other step)

File: docs/kubos-linux-on-iobc.md (hunk 3 above)
Comment (CatFreed) [inline docs/kubos-linux-on-iobc.md]: :+1:

**Pull request merged**